pkg/util/utils.go

```diff
@@ -37,6 +37,12 @@ import (
 	"k8s.io/minikube/pkg/version"
 )
 
+type RetriableError struct {
+	Err error
```

Comment:
I ran into some trouble trying to embed the builtin error interface and referencing across packages.

I ended up just keeping this how it was, but adding some additional context to the error message.